tst/I18nTest.php

```diff
@@ -167,6 +185,38 @@ public function testHtmlEntityEncoding()
         $this->assertEquals($result . $result, I18n::_($input . '%s', $input), 'encodes message ID as well, when no link');
     }
 
+    public function testFallbackAlwaysPresent()
+    {
+        $path = sys_get_temp_dir() . DIRECTORY_SEPARATOR . 'privatebin_i18n';
+        if (!is_dir($path)) {
+            mkdir($path);
+        }
+
+        $languageIterator = new AppendIterator();
+        $languageIterator->append(new GlobIterator(I18nMock::getPath('??.json')));
+        $languageIterator->append(new GlobIterator(I18nMock::getPath('???.json'))); // for jbo
+        $languageCount = 0;
+        foreach ($languageIterator as $file) {
+            ++$languageCount;
+            $this->assertTrue(copy($file, $path . DIRECTORY_SEPARATOR . $file->getBasename()));
+        }
+
+        I18nMock::resetPath($path);
+        $languagesDevelopment = I18nMock::getAvailableLanguages();
+        $this->assertEquals($languageCount, count($languagesDevelopment), 'all copied languages detected');
+        $this->assertTrue(in_array('en', $languagesDevelopment), 'English fallback present');
+
+        unlink($path . DIRECTORY_SEPARATOR . 'en.json');
```

Comment:
I would split it into two test ~~classes~~ functions (also to keep it readable and keep the mantra "one test per function").

But yeah…